docs/src/main/asciidoc/spring-cloud-config.adoc

```diff
@@ -1361,7 +1361,7 @@ Spring Retry has a `RetryInterceptorBuilder` that supports creating one.
 
 === Locating Remote Configuration Resources
 
-The Config Service serves property sources from `/{name}/{profile}/{label}`, where the default bindings in the client app are as follows:
+The Config Service serves property sources from `/{application}/{profile}/{label}`, where the default bindings in the client app are as follows:
```

Comment:
Using "sources" as a verb made me read this multiple times to figure it out. How about replacing "sources from" with "derives from"?